website/write_ar_table.php

```diff
@@ -310,4 +310,79 @@ function write_new_ar_table($date)
 		print("	</div>\n") ;
 		print("</div>\n");			
 	}
+
+
+
+
+
+	function write_new_ch_table($date)
+	{	
+	include ("globals.php");
+		
+		//	Contruct the file name
+		$file = "${arm_data_path}data/" . $dirdate . "/meta/arm_ch_summary_" . $date . ".txt";
+
+			print("<div class=noaat>\n");		
+			print("<table class='frame' rules=rows width=700 align=center cellpadding=0 cellspacing=0 frame=hsides>\n");
+			print("	  <tr align=center class=noaatit>\n");
+			print("         <td colspan=6> Today's Coronal Hole Properties </td>\n");
+			print("   </tr>\n");
+			print("   <tr align=center class=noaacolumns>\n");
+			print("         <td class=noaacol><i><div onmouseover=\"title='This is a unique number assigned to each new coronal hole by CHIMERA.'\">CHIMERA Number</div></i></td>\n");
+			print("         <td class=noaacol><i><div onmouseover=\"title='The centroids are given in heliocentric (arcseconds from Sun centre).'\">Centroid <br> [Heliocentric]</a></div></td>\n");
+			print("         <td class=noaacol><i><div onmouseover=\"title='The logitudinal width of CHs for forecasting arrival of solar wind'\">Width <br> [&deg;]</div></i></td>\n");
+			print("         <td class=noaacol><i><div onmouseover=\"title='The area in % of the solar disk area'\">Area <br> [%]</div></i></td>\n");
+			print("         <td class=noaacol><i><div onmouseover=\"title='The mean magnetic field in the CH boundary.'\"> < B > <br> [G]</div></i></td>\n");
+			print("         <td class=noaacol><i><div onmouseover=\"title='The mean magnetic flux in the CH boundary'\"> < &Phi; > <br> [Mx]</div></i></td>\n");
+			print("   </tr>\n");
+		
+		//	Print the start of the table and the column headers.  These always display.
+//TODO: this if should go before the header or make a change to say that there is not ARs			
+		if (file_exists($file))
+		{
+			//	Read the entire contents of the file in to the lines array
+
+			$lines = file($file);
+			$linen = 0;
+			foreach ($lines as $line)
+			{
+				//	Extract all info from the line.  Events that get hyperlinks are all stored in $events and need to be split later.
+				list($number, $xcen, $ycen, $xeb, $yeb, $xwb, $ywb, $xnb , $ynb, $xsb, $ysb, $width, $area1, $area2, $mB, $mBpos, $mBneg, $Bmax, $Bmin, $TotBPos, $TotBNeg, $Phi, $Phipos, $Phineg) = preg_split('/\s+/', $line, 23);
+
+				if("$linen" > "1")

```

Comment:
I think this is a bit obfuscated... are you just skipping the first line of the file? I would just do the opposite comparison, to avoid extra indentation.